packages/dd-trace/src/plugins/util/web.js

```diff
@@ -426,13 +493,13 @@ function addRequestTags (context) {
   const { req, span, config } = context
   const url = extractURL(req)
 
-  span.addTags({
+  span.addTags(Object.assign({
```

Comment:
Is a developer decision or exist some convention to use `Object.assign` instead of:
```
  span.addTags({
    [HTTP_URL]: web.obfuscateQs(config, url),
    [HTTP_METHOD]: req.method,
    [SPAN_KIND]: SERVER,
    [SPAN_TYPE]: WEB,
    [HTTP_USERAGENT]: req.headers['user-agent'],
    ... web.extractIp(context)
}
```

Reply:
If I remember correctly the spread operator is awfully slow, and it might not be supported on old runtimes. But I might be wrong here.

Reply:
I haven't benchmarked it, so probably you are right about the performance.

But it looks like it is compatible since node `8.5.3`